packages/stripe_platform_interface/lib/src/models/payment_sheet.dart

```diff
@@ -106,6 +106,13 @@ class SetupPaymentSheetParameters with _$SetupPaymentSheetParameters {
     /// The list of preferred networks that should be used to process payments made with a co-branded card.
     /// This value will only be used if your user hasn't selected a network themselves.
     @JsonKey(toJson: _cardBrandListToJson) List<CardBrand>? preferredNetworks,
+
+    /// set the payment sheet language
+    String? languageCode,
```

Comment:
We need to provide the interface provided by the stripe sdk in react native and as you can see it does not have language code yet: https://github.com/stripe/stripe-react-native/blob/master/src/types/PaymentSheet.ts

Reply:
I understand, this was more of a feature that I added since a language force is possible on the web but not on the mobile platforms.